### ISSUE TITLE
Rewrite MPI Section of Website

### ISSUE THREAD

**Issue by ellestad** [ucsf-wynton/wynton-website-hpc#138]
I believe these versions of OpenMPI are no longer available in Rocky 8.
```
OpenMPI 3.1.3, available via module load mpi or module load mpi/openmpi3-x86_64
OpenMPI 1.10.7, available via module laod mpi/openmpi-x86_64
```
Only `openmpi-4.1.1-3.el8.x86_64` is available.

Rewrite this section:

https://wynton.ucsf.edu/hpc/scheduler/submit-jobs.html#mpi-single-host-parallel-processing--pe-mpi_onehost-slots

**Comment by benmwebb** [comment]
(Also, "laod" is misspelled!)

**Comment by HenrikBengtsson** [comment]
Fixed (commit 4d6fa2d).
